resource_booking/models/calendar_event.py

```diff
@@ -51,3 +51,13 @@ def write(self, vals):
                 rescheduled -= new
         rescheduled._validate_booking_modifications()
         return result
+
+    def get_interval(self, interval, tz=None):
+        """Autofix tz from related resource booking.
+
+        This function is called to render calendar.event notification mails.
+        Any notification related to a resource.booking must be emitted in the
+        same TZ as the resource.booking. Otherwise it's confusing to the user.
+        """
+        tz = self.resource_booking_ids.type_id.resource_calendar_id.tz or tz
+        return super().get_interval(interval, tz)
```

Comment:
Keep keyword arguments as keyword, not positional

Reply:
Thanks for the suggestion, but I don't see any advantages.

Reply:
It's not a suggestion, it's a Python guideline.

Reply:
Done.